code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -79,6 +79,66 @@ func (ssc *StorageSmartContract) GetBlobberHandler(
 	return sn, err
 }
 
+// GetBlobberCountHandler returns Blobber count from its individual stored value.
+func (ssc *StorageSmartContract) GetBlobberCountHandler(
+	ctx context.Context,
+	params url.Values,
+	balances cstate.StateContextI,
+) (resp interface{}, err error) {
+	blobberCount, err := balances.GetEventDB().GetBlobberCount()
+	if err != nil {
+		return nil, fmt.Errorf("Error while geting the blobber count")
+	}
+	return map[string]int64{
+		"count": blobberCount,
+	}, nil
+}
+
+// GetBlobberTotalStakesHandler returns blobber total stake
+func (ssc *StorageSmartContract) GetBlobberTotalStakesHandler(
+	ctx context.Context,
+	params url.Values,
+	balances cstate.StateContextI,
+) (resp interface{}, err error) {
+	blobbers, err := balances.GetEventDB().GetBlobbers()
```

Comment:
we should implement method to get only specific fields when needed
here we can save getting just blobberID over the network than the entire data